### PR TITLE
Translatewiki: makefile, model, tuning report, and utilities.

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1 @@
-models/*.model filter=lfs diff=lfs merge=lfs -text
+*.model filter=lfs diff=lfs merge=lfs -text
```
Comment (adamwight) [inline .gitattributes:1]: The syntax seems right, but why was this necessary?

File: Makefile.manual [hunk 2]
```diff
@@ -300,6 +300,77 @@ plwiki_tuning_reports: \
...
+		datasets/translatewiki.autolabeled_revisions.2018.json
+	(cat $< | grep '"reverted_for_damage": true'; \
+	 cat $< | grep '"reverted_for_damage": false' | shuf -n 20000) > $@
+
+datasets/translatewiki.stratified_revisions.w_cache.25k_2018.json: \
+		datasets/translatewiki.stratified_revisions.25k_2018.json
+	cat $< | \
+	revscoring extract \
+		editquality.feature_lists.translatewiki.reverted \
+		editquality.feature_lists.translatewiki.damaging \
```
Comment (adamwight) [inline Makefile.manual:330]: Is this just a courtesy for later?

**Review by adamwight — COMMENTED** [review]
> Git LFS file not shown
Great work!  Notes for follow-up, you were going to update git-lfs on stat1007 maybe, and it's worth documenting any steps in the LFS workflow which might give us trouble in the future.

**Comment by adamwight** [comment]
Needs lint, otherwise I would merge.

File: Makefile.manual (hunk 2 above)
Comment (halfak) [reply Makefile.manual:330]: Yeah.  It's a standard practice for our past models.

File: .gitattributes (hunk 1 above)
Comment (halfak) [reply .gitattributes:1]: That's a good question.  I have no idea.  But it did work.

**Comment by adamwight** [comment]
Handful more trivial lint fixes:
```
./editquality/utilities/autolabel.py:204:54: E502 the backslash is redundant between brackets
./editquality/utilities/autolabel.py:210:53: E502 the backslash is redundant between brackets
```
